### ISSUE TITLE
not compatible with eslint 5: Cannot find module 'eslint/lib/ast-utils'

### ISSUE THREAD

**Issue by srghma** [alecxe/eslint-plugin-protractor#93]
have an error

```
Error: Failed to load plugin 'protractor' declared in '.eslintrc.js': Cannot find module 'eslint/lib/ast-utils'
```

how to fix https://github.com/eslint-plugin-cleanjs/eslint-plugin-cleanjs/issues/3#issuecomment-497969757

version 

```
eslint@^5.0.0:
  version "5.16.0"
eslint-plugin-protractor@^1.41.1:
  version "1.41.1"
```

**Comment by srghma** [comment]
fixed by upgrading to master

"eslint-plugin-protractor": "git+https://github.com/alecxe/eslint-plugin-protractor.git#master",

**Comment by alecxe** [comment]
Thanks for flagging this. I've noticed an issue on travis and with my github setup which led to semantic-release failing to publish. I've fixed the issue and forced-published 2.0.0 - also thought it'd be a good time to freshen things up in versioning :)
